addons/website_event/views/event_templates_widgets.xml

```diff
@@ -2,6 +2,24 @@
 <odoo>
 
 <!-- Search Box -->
+<template id="events_search_box_input" name="Events search box">
+    <t t-call="website.website_search_box_input">
+        <t t-set="_classes" t-value="'o_wevent_event_searchbar_form w-100 my-1 my-lg-0 %s' % _classes"/>
+        <t t-set="search_type" t-valuef="events"/>
```

Comment:
Why use valuef in the second but not in the first?

Reply:
The first one combines the old value of `_classes` with a hard-coded set using `%`, the second is a plain string - and so far I have been told to prefer `t-valuef` for non-computed values.

Reply:
> so far I have been told to prefer t-valuef for non-computed values.

Sure, maybe, to avoid translating them I guess, but the actual original purpose of `valuef` is to use format-string features in t-value computations. Which is exactly what the first wants / is doing.

Reply:
Oh... therefore the "f"